sections/memory_safety_in_rust.tex

```diff
@@ -35,6 +32,29 @@ \section{Memory Safety in Rust}
 
 \end{frame}
 
+
+%%%
+
+\begin{frame}[fragile]{Ergonomic scaffolding with \texttt{unwrap()}}
+Explicit error handling may seem like a huge barrier.
+
+Extracting values from \texttt{Option<T>}, \texttt{Result<T, E>} and may others fast with \texttt{unwrap()} and abort the program on failure:
```

Comment:
Also maybe add: "`expect("error message")` does the same thing but customizes the error message"